libs/remix-ui/git/src/components/gitui.tsx

```diff
@@ -200,37 +200,38 @@ export const GitUI = (props: IGitUi) => {
                     <hr></hr>
                     <BranchesNavigation eventKey="2" activePanel={activePanel} callback={setActivePanel} />
                     <Accordion.Collapse className='bg-light' eventKey="2">
-                      <>
-                        <Branches /></>
+                      <div className="px-2 py-2">
+                        <Branches />
+                      </div>
                     </Accordion.Collapse>
                     <hr></hr>
                     <RemotesNavigation eventKey="5" activePanel={activePanel} callback={setActivePanel} />
                     <Accordion.Collapse className='bg-light' eventKey="5">
-                      <>
-                        <Remotes></Remotes>
-                      </>
+                      <div className="px-2 py-2">
+                        <Remotes plugin={plugin}></Remotes>
```

Comment:
don't do this. it's not needed. the plugin is in the context if you need to call it.

Reply:
done